server/uploadHandler/src/service/dynamodbClient.ts

```diff
@@ -0,0 +1,44 @@
+import { AWSError, DynamoDB } from "aws-sdk";
+import {
+  GetItemInput,
+  GetItemOutput,
+  PutItemInput,
+  PutItemOutput,
+  QueryInput,
+  QueryOutput,
+  UpdateItemInput,
+  UpdateItemOutput,
+} from "aws-sdk/clients/dynamodb";
+import { PromiseResult } from "aws-sdk/lib/request";
+import { Agent } from "https";
+
+const awsRegion = process.env.AWS_REGION
+
+// The reason of using DynamoDB directly, see:
+// https://github.com/aws/aws-sdk-js/issues/3578
```

Comment:
I'm unsure how that GitHub issue relates to this code.

Reply:
Commets Removed.